src/index.js

```diff
@@ -1,93 +1,100 @@
-const fs = require("fs"),
-    http = require("http"),
-    path = require("path"),
-    methods = require("methods"),
-    express = require("express"),
-    bodyParser = require("body-parser"),
-    session = require("express-session"),
-    cors = require("cors"),
-    passport = require("passport"),
-    errorhandler = require("errorhandler"),
-    mongoose = require("mongoose");
-
-const isProduction = process.env.NODE_ENV === "production";
+const fs = require('fs'),
```

Comment:
Babel is already configured to work with this project. So, please change this code to use import from ES6 instead of require. For example: `import fs from 'fs'`